src/database/schema.zmodel

```diff
@@ -797,3 +804,178 @@ model LastKeycloakUpdate {
 
   @@map("last_keycloak_update")
 }
+
+
+
+// SHOP SYSTEM
+
+enum ShoppableType {
+  TICKET
+}
+
+// Base entity to be inherited by actual purchasables
+model Shoppable {
+  id String @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  title String
+  titleEn String?
+  description String
+  descriptionEn String?
+  type ShoppableType
+
+  authorId String @db.Uuid @deny("update", true) // never allow updating wh created the shoppable
+  author Member @relation(fields: [authorId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+
+  price Int
+  availableFrom DateTime @map("available_from") @db.Timestamptz(6) @default(now())
+  availableTo DateTime? @map("available_to")
```

Comment:
Why no `@db.Timestamptz(6)` here? Am I missing something?

Reply:
Good catch. Honestly, the only reason we have `@db.Timestamptz(6)` everywhere is due to it being imported from the old database. I don't really think it is, or isn't, necessary. But I guess we could be consequential.